start.py

```diff
@@ -92,8 +92,7 @@ def main(args):
             node.volumes.append(java_image)
 
     if args.spark2_version:
-        logger.info('args.spark2_version = %s', args.spark2_version)
-        spark2_parcel_image = ('{}/{}/clusterdock:cdh-spark-{}').format(args.registry,
+        spark2_parcel_image = ('{}/{}/clusterdock:cdh_spark_{}').format(args.registry,
```

Comment:
Sorry to be annoying, but let's have this `clusterdock:topology_cdh-spark2-{}` to be consistent with how we currently name images of things like SDC